### PR TITLE
docs: add a task for weighted TCP routing

### PR DESCRIPTION
**Pull request by venilnoronha**
This adds a new Task under the Traffic Management module for showcasing Istio's Weighted TCP Routing feature.

/cc @andraxylia @rshriram

### CODE REVIEW

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+title: TCP Traffic Shifting
+description: Shows you how to migrate TCP traffic from an old to new version of a TCP service.
+weight: 25
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]: change the weight to 26 so it comes after the http traffic shifting task

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/tcp-echo/tcp-echo-all-v1.yaml@
+    {{< /text >}}
+
+1.  Confirm that the `tcp-echo` service is up and running.
+
+    The `$INGRESS_HOST` variable below is the External IP address of the ingress, as explained in
+the [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port) doc. To obtain the
+`$INGRESS_PORT` value, use the following command.
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]: Why not just tell user to set the INGRESS_HOST and INGRESS_PORT variables according to [these instructions](https://istio.io/docs/tasks/traffic-management/ingress/#determining-the-ingress-ip-and-ports)?

Comment (venilnoronha) [reply content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]: The document you mentioned only speaks about `http2` and `https`, while we're working with the `tcp` port here.

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md (hunk 1 above)
Comment (venilnoronha) [reply content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]: Addressed in 2909c94.

**Pull request merged**